collectors/ebpf.plugin/ebpf.d.conf

```
@@ -57,7 +57,8 @@
     oomkill = yes
     process = yes
     shm = yes
-    socket = no # Disabled while we are fixing race condition
+    # 'socket' is disabled while we are fixing race condition (https://github.com/netdata/netdata/issues/12027)
+    socket = no
```

```suggestion
    # 'socket' is disabled while we are fixing race condition (https://github.com/netdata/netdata/issues/12027)
    # socket = no
```

@thiagoftsm
or something like this if the goal was to (kind of) have no "socket" section, so later we can:
 - re-enable it 
 - and users who created user 'ebpf.d.conf' will get it re-enabled too

---

I already discussed with product managers about a section for socket, we are expecting to have it as soon blockers are removed.

---

> or something like this 

ok, that won't work, we're loading one file (not merging user/stock).

---

For your specific PR, you can move the comment above and keep `socket = no`, this is enough to disable the thread.

The fact that socket will have another section, it is a completely different subject.